.github/workflows/ts_test.yaml

```diff
@@ -48,5 +48,34 @@ jobs:
       - name: Generate Proof
         run: nargo prove --package main --oracle-resolver=http://localhost:5555
 
+      - name: Run nargo codegen-verifier
+        run: nargo codegen-verifier --package main
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: Start Anvil
+        run: |
+          anvil &
+          ANVIL_PID=$!
+          echo "ANVIL_PID=$ANVIL_PID" >> $GITHUB_ENV
+
+      - name: Compile Smart Contract
+        working-directory: contracts
+        run: forge build
+
+      - name: Deploy Smart Contract
+        working-directory: contracts
+        run: forge create --rpc-url $ETH_RPC_URL --private-key $PRIVATE_KEY UltraVerifier
+        env:
+          ETH_RPC_URL: '127.0.0.1:8545'
+          PRIVATE_KEY: '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80'
```

Comment:
This repo is public and even though this "private key" is in fact public - it still looks bad. Let's add it as a hidden env var through github to build good practices

Reply:
Renamed to ANVIL_TEST_ACCOUNT_PRIVATE_KEY. This is deterministic test account private key. For opensource lib we want anybody to fork this repo and run all tests without additional configuration. Hiding this (in fact publicly available data) would prevent this github action to pass out-of-the-box.